apps/explorer/src/index.css

```diff
@@ -25,29 +25,6 @@ body {
     @apply m-0 text-offblack;
 }
 
-html {
```

Comment:
The below scrollbar CSS was created to follow the Figma Design.

Reply:
It is unclear what the exact deltas between this scrollbar CSS and the updated Figma Design are, but if they do exist, the efficient way forward would be to implement what few minor changes are required on this existing code. This would quickly resolve issue https://github.com/MystenLabs/sui/issues/3938.

Reply:
This was discussed at length internally here:  https://mysten-labs.slack.com/archives/C03RUGM5J8P/p1660243535444779
This was the outcome we agreed on.